frontend/src/scenes/plugins/plugin/PluginCard.tsx

```diff
@@ -130,7 +130,7 @@ export function PluginCard({
                                     <GlobalOutlined /> Managed by {organization_name}
                                 </Tag>
                             )}
-                            {user?.organization?.plugins_access_level === PluginsAccessLevel.Root && (
+                            {user?.organization?.id === organization_id && (
```

Comment:
Shouldn't this still check the access level as well?

Reply:
It doesn't have to, `organization_id` has to be from an org which has at least install access by definition.